lotsqrl/camera.py

```diff
@@ -48,20 +48,20 @@ def load_sprites(self):
         if not self.options.graphical_tiles:
             return
 
-        terminal.set("tile 0x28: graphics\\cocoon.png, size=16x16, spacing=2x1;")
-        terminal.set("tile 0x30: graphics\\egg.png, size=16x16, spacing=2x1;")
-        terminal.set("tile 0x67: graphics\\goblin.png, size=16x16, spacing=2x1;")
-        terminal.set("tile 0x47: graphics\\goblin-chief.png, size=16x16, spacing=2x1")
-        terminal.set("tile 0x2e: graphics\\floor2.png, size=16x16, spacing=2x1;")
-        terminal.set("tile 0x25: graphics\\skull.png, size=16x16, spacing=2x1;")
-        terminal.set("tile 0x53: graphics\\spider.png, size=16x16, spacing=2x1;")
-        terminal.set("tile 0x73: graphics\\spiderling.png, size=16x16, spacing=2x1;")
-        terminal.set("tile 0x40: graphics\\spiderqueen.png, size=16x16, spacing=2x1;")
-        terminal.set("tile 0x23: graphics\\wall2.png, size=16x16, spacing=2x1;")
-        terminal.set("tile 0x2f: graphics\\webdiagonal2.png, size=16x16, spacing=2x1")
-        terminal.set("tile 0x5c: graphics\\webdiagonal.png, size=16x16, spacing=2x1")
-        terminal.set("tile 0x2d: graphics\\webhorizontal.png, size=16x16, spacing=2x1")
-        terminal.set("tile 0x7c: graphics\\webvertical.png, size=16x16, spacing=2x1")
+        terminal.set(f"tile 0x28: {self.options.graphics_base_dir/'cocoon.png'}, size=16x16, spacing=2x1;")
```

Comment:
We could avoid some pasta here using tuples of  (hex, image name) then iterating upon it to call terminal.set
Would allow us to join the path and make the entire code more compact